workshops/spinning_wheel/README.md

```diff
@@ -0,0 +1,434 @@
+
+# Part 1: Prerequisites
+* Account on Repl.it
+* Basic knowledge of HTML, CSS and JavaScript would be helpful but is not necessary to continue further.
+
+# Part 2: Setup
+## Setting up our code environment
+We can use any code editor but keeping accessibility in our mind, we will be using Repl.it. It is an online code editor that is easy to use and can directly put your website in the real world.
+
+You can get started with it by going to [repl.it/languages/html](https://repl.it/languages/html). You will directly land on the page where we can write our code.
+
+![Default Code](https://cloud-2un6g11k6.vercel.app/0default-file.png)
+
+Right now your page will look like this.
+# Part 3: Inspecting The Default Files 
+Here on the right side in the files section, you can see 3 files that are:
+1. index.html
+2. script.js
+3. style.css.
+
+Now let’s take a look at those and try to understand it individually.
+
+## 1)	The HTML file
+HTML gives the structure to our webpage. Here in the first line, we have `<!DOCTYPE html>` which is used for specifying the version of HTML the document is using. Next, we have the `<title>` tag which specifies the title of our webpage that will be shown on the tab bar. Then we have <body> tag which contains the main visible part of our web page. Make sure that all the code you write is included between the opening and closing tags. ( `<body></body>` )
+  
+  Make sure to remove line 5 (`<meta name="viewport" content="width=device-width">`) which may cause some weird responsiveness issues for our workshop.
+  
+  ![Removing the fifth line](https://cloud-2un6g11k6.vercel.app/2html-1.gif)
+  
+## 2)	The CSS file
+This is the style.css file in your directory. This defines the looks of our website. Whatever styles we give to our website is written in this file. If you take a look in the <head> tag in your HTML file, you will find a line of code (`<link href="style.css" rel="stylesheet" type="text/css" />`). This means that your HTML file is properly linked to your CSS file!
+
+## 3)	The JavaScript file
+This is the script.js file in your directory. It's empty for now, but we will fill it eventually through the workshop! The code written here will be responsible for the functioning of our Wheel. If you take a look in the <body> tag in your HTML file, you will find a line of code (`<script src="script.js"></script>`). This means that your HTML file is properly linked to your JavaScript file!
+
+# Part 4: Building the Wheel
+
+## 1)	HTML
+
+Let’s begin the coding part of our workshop. Here we will write the HTML code required to build our wheel.
+
+Give a division with the help of `<div>` tag inside our body tags. Now give `id="mainbox"` `class="mainbox"` in that division, by this CSS and JavaScript file can communicate for the specific division (`<div id="mainbox" class="mainbox"> </div>` should be the final product for this. 
```

Comment:
Weird phrasing "Give a division". I think division isn't quite the right word here, even though that's what <div> means. Maybe say something like "section"? Or another word that describes it better. The grammar/phrasing of this paragraph is also a little weird, which makes it a little difficult to read